shin-sforzando/yomutube
# feat: :+1: build infrastructure using Pulumi

## Describe the PR

To close #2

## Screenshots

![pulumi](https://github.com/shin-sforzando/yomutube/assets/32637762/5206c68f-f342-4a83-97cd-98934277c594)

## Detail of the change

- [x] Introduce Pulumi
- [x] Introduce pre-commit
- [x] Introduce git-secret

## Anticipated impacts

None.

## To reproduce

Steps to check the behavior:

1. `make setup`
1. Change directory to `pulumi`
1. `make up`
1. Check on [pulumi.cloud](https://app.pulumi.com/shin-sforzando/YomuTube/dev)

## Appendix

None.
